components/raftstore-v2/src/batch/store.rs

```diff
@@ -142,7 +167,11 @@ impl<EK: KvEngine, ER: RaftEngine, T: Transport + 'static> PollHandler<PeerFsm<E
             HandleResult::stop_at(0, false)
         };
         let mut delegate = PeerFsmDelegate::new(peer, &mut self.poll_ctx);
-        delegate.handle_msgs(&mut self.peer_msg_buf);
+        delegate.on_msgs(&mut self.peer_msg_buf);
+        delegate
+            .fsm
+            .peer_mut()
+            .handle_raft_ready(delegate.store_ctx);
```

Comment:
poll_ctx, store_ctx are essentially same object. Can we use the poll_ctx as it's widely used in the code.

Reply:
No, they are not. There maybe some common fields, those parts can be extracted.

Reply:
I mean just in this function, we have two names for the same thing:
Here it's called poll_ctx.
let mut delegate = PeerFsmDelegate::new(peer, &mut self.poll_ctx);
But in PeerFsmDelegate, it's called store_ctx.